src/gui/qgisinterface.h

```diff
@@ -327,6 +327,11 @@ class GUI_EXPORT QgisInterface : public QObject
      */
     virtual QMenu *webMenu() = 0;
 
+    /**
+     * Returns a reference to the main window "Mesh" menu.
+     */
```

Comment:
```suggestion
    /**
     * Returns a reference to the main window "Mesh" menu.
     *
     * \since QGIS 3.36
     */
```

A backport would be nice (at least 3.34)
Adapt since depending on backport or not ?

Reply:
OK for 3.34